src/Tickets/Emails/Hooks.php

```diff
@@ -0,0 +1,61 @@
+<?php
+/**
+ * Handles hooking all the actions and filters used by the module.
+ *
+ * To remove a filter:
+ * remove_filter( 'some_filter', [ tribe( TEC\Tickets\Emails\Hooks::class ), 'some_filtering_method' ] );
+ * remove_filter( 'some_filter', [ tribe( 'tickets.emails.hooks' ), 'some_filtering_method' ] );
+ *
+ * To remove an action:
+ * remove_action( 'some_action', [ tribe( TEC\Tickets\Emails\Hooks::class ), 'some_method' ] );
+ * remove_action( 'some_action', [ tribe( 'tickets.emails.hooks' ), 'some_method' ] );
+ *
+ * @since   TBD
+ *
+ * @package TEC\Tickets\Emails
+ */
+
+namespace TEC\Tickets\Emails;
+
+use \tad_DI52_ServiceProvider;
+
+/**
+ * Class Hooks.
+ *
+ * @since   TBD
+ *
+ * @package TEC\Tickets\Emails
+ */
+class Hooks extends tad_DI52_ServiceProvider {
```

Comment:
I don't think we need this as a service provider. It can be a plain class.

Reply:
Spoke to Gustavo about this. Hooks and Assets need to be loaded up front, therefore they will need to be loaded as service providers. You were right about the Emails_Tab class, though. It needs to be a regular class.